.github/workflows/changelog.yml

```diff
@@ -0,0 +1,93 @@
+# This action requires that any PR targeting the main branch should add a
+# yaml file to the ./.chloggen/ directory. If a CHANGELOG entry is not required,
+# or if performing maintance on the Changelog, add either \"[chore]\" to the title of
+# the pull request or add the \"Skip Changelog\" label to disable this action.
+
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
Based on previous conversations, I think the agreement is to render change log entry into `CHANGELOG-AWS.md` when a PR is merged. 

Looks like this workflow checks if a changelog entry is added on PR creation, but is there a workflow to automatically convert the entry post-merge or am I missing something?